vm-allocator/src/page.rs

```diff
@@ -0,0 +1,38 @@
+// Copyright 2023 Arm Limited (or its affiliates). All rights reserved.
```

Comment:
I am not sure it's good to put this file in `vm-allocator`, because it really has nothing with memory allocation.

If there is not any other potential consumer of the functions in this file, how about moving them to `vfio.rs`?

Reply:
These tools should not belong to any module as it's common and need to be accessed by the whole project. I find no better place to hold them. I put them here becuse it's the first place to refer to getting page size.

Reply:
OK. Thanks.